vernac/declare.ml

```diff
@@ -1084,13 +1077,9 @@ let replace_appvars subst =
   Constr.map aux
 
 let subst_prog subst prg =
-  if get_hide_obligations () then
-    ( replace_appvars subst prg.prg_body
```

Comment:
CI is failing because this was the only place this function was used, so it should be removed as well.